packages/beta/src/types.ts

```diff
@@ -57,6 +57,19 @@ export const MonitoringTaskModelExternalId = {
   DOUBLE_THRESHOLD: 'double_threshold' as const,
 };
 
+export type AlertStatus = 'FIRING' | 'PENDING';
+
+export const AlertStatus = {
+  FIRING: 'FIRING' as AlertStatus,
+  PENDING: 'PENDING' as AlertStatus,
+};
+
+export interface StartTime {
+  min?: number;
+  max?: number;
+
+};
+
```

Comment:
no need of this type, use DateRange;
```suggestion
```

Reply:
ok, thanks! :D just trying my best here :D